src/commands/generateLockFiles.js

```diff
@@ -1,11 +1,11 @@
 const {kebabCase} = require('lodash')
 
-const {temporaryTransformPackage} = require('./lib/temporaryTransformPackage')
-const {runNpmInstall} = require('./lib/runNpmInstall')
+const {temporaryTransformPackage} = require('./utils/temporaryTransformPackage')
+const {runNpmInstall} = require('./utils/runNpmInstall')
 
 const path = require('path')
 
-function generateLockFiles({basedir}) {
+function generateLockFiles(basedir) {
```

Comment:
Changing from `{basedir}` to `basedir` makes the CLI command `sanity-template lockfiles` fail:

```
The "path" argument must be of type string. Received an instance of Object
    at validateString (internal/validators.js:117:11)
    at Object.join (path.js:1039:7)
```